deepmd/cmake/Findtensorflow.cmake

```diff
@@ -10,32 +10,55 @@
 # TensorFlowFramework_LIBRARY    
 # TensorFlowFramework_LIBRARY_PATH
 
+
+if(NOT DEFINED TENSORFLOW_ROOT)
+  message(STATUS "TENSORFLOW_ROOT not set, finding in current python environment...")
+  set(FIND_TENSORFLOW_ROOT_CMD "import os, tensorflow; print(os.path.dirname(tensorflow.__file__), end='')")
+  execute_process(
+          COMMAND "python" "-c" "${FIND_TENSORFLOW_ROOT_CMD}"  # Automatically set PYTHON_EXECUTABLE
+          WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
+          OUTPUT_VARIABLE TENSORFLOW_ROOT
+          RESULT_VARIABLE TENSORFLOW_ROOT_RESULT_VAR
+          ERROR_VARIABLE TENSORFLOW_ROOT_ERROR_VAR
+  )
+  message(STATUS "Set hypothetical TENSORFLOW_ROOT as: ${TENSORFLOW_ROOT}")
+endif()
+
 string(REPLACE "lib64" "lib" TENSORFLOW_ROOT_NO64 ${TENSORFLOW_ROOT})
 
+
 # define the search path
 list(APPEND TensorFlow_search_PATHS ${TENSORFLOW_ROOT})
+list(APPEND TensorFlow_search_PATHS "/home/lzhpc/WORK/caizefeng/soft/")
```

Comment:
What's this line?

Reply:
My bad, forget to delete the test code.  Just forget this PR anyway.